server/tests/integration/test_api.ts

```diff
@@ -0,0 +1,106 @@
+import { agent } from "supertest";
+import { app } from "../../src";
+import { expect } from "chai";
+import { restoreDb } from "../../db/restore_db";
+import { seedDb } from "../../db/seed_db";
+
+describe("test_controller.ts API", () => {
+    const patient5Token =
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjUsImlhdCI6MTY0NTEzNzk0Mn0.UcpnGpWBhdM4OEgKOrJEWXoknk9-I_2Cf19pJWkS5eY";
+    const doctor1Token =
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjEsImlhdCI6MTY0NTk5NzI4Mn0.Is7O_xeaFrijN4uyM6tKDP7W9yzYabbugScC1w2Kn-s";
+    const healthOfficialToken =
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOjcsImlhdCI6MTY0NTk5NzA4OX0.c2zU_Z7lNO0vjX4dAE0Ara_LSP0YlIV-heWPa5kAQEs";
```

Comment:
There is a token fixture file containing all the tokens you should import instead of hard coding them in each test